Cargo.toml

```diff
@@ -86,23 +86,24 @@ fuel-abi-types = "0.7"
 # Although ALL verions are "X.Y", we need the complete semver for
 # fuel-core-client as the GitHub Actions workflow parses this value to pull down
 # the correct tarball
-fuel-core-client = { version = "0.36.0", default-features = false }
-fuel-core-types = { version = "0.36", default-features = false }
+fuel-core-client = { version = "0.37", default-features = false }
+fuel-core-types = { version = "0.37", default-features = false }
 
 # Dependencies from the `fuels-rs` repository:
-fuels = "0.66"
-fuels-core = "0.66"
-fuels-accounts = "0.66"
+
+fuels = "0.66.6"
+fuels-core = "0.66.6"
+fuels-accounts = "0.66.6"
 
```

Comment:
Should this be reverted since we're decided to go with "X.Y" rather than "X.Y.Z" releases?